pkg/loadinfo/loadinfo.go

```diff
@@ -85,8 +86,8 @@ func LogCurrentSystemLoad(logFunc LogFunc) {
 						toMB(memInfo.Stack), toMB(memInfo.Locked), toMB(memInfo.Swap))
 				}
 
-				logFunc("NAME %s STATUS %s PID %d CPU: %.2f%% MEM: %.2f%% %s",
-					name, status, p.Pid, cpuPercent, memPercent, memExt)
+				logFunc("NAME %s STATUS %s PID %d CPU: %.2f%% MEM: %.2f%% CMDLINE: %s MEM-EXT: %s",
```

Comment:
Can we print the CMDLINE at the end? Otherwise we'll have to scroll right to get the mem-ext value

Reply:
I did this on purpose. I found the CMDLINE to be more important as the extended memory statistics. The memory percentage is right there at the beginning.